android/src/main/java/com/reactnativecommunity/asyncstorage/AsyncStoragePackage.java

```diff
@@ -27,7 +26,7 @@
         if (BuildConfig.AsyncStorage_useNextStorage) {
             try {
                 Class storageClass = Class.forName("com.reactnativecommunity.asyncstorage.next.StorageModule");
-                NativeModule inst = (NativeModule) storageClass.getDeclaredConstructor(new Class[]{ReactContext.class}).newInstance(reactContext);
+                NativeModule inst = (NativeModule) storageClass.getDeclaredConstructor(new Class[]{ReactApplicationContext.class}).newInstance(reactContext);
```

Comment:
Do you mind explaining how this fixes the issue? We're not explicitly use app context within next storage, maybe somewhere implicit? How did you come to this conclusion? 🙏